src/core/updateWebpackConfig.js

```diff
@@ -28,7 +28,11 @@ const updateWebpackConfig = (webpackConfig, env, serverInfo) => {
     }
   )
 
-  webpackConfig.entry.push(stylesPath('styles.css'))
+  webpackConfig.entry.push(
+    stylesPath('styles.css'),
+    'codemirror/lib/codemirror.css',
+    'rsg-codemirror-theme.css'
```

Comment:
That's a good point, thanks for raising it.
I thought we could avoid importing the CSS on our side as well.

Unfortunately, "Removing the imports doesn't change the visual output" is true only if you click "Show code" before expanding the diff.
If you expand the diff without showing the code, it doesn't work (and that's the reason we are importing CodeMirror's files):

<img width="386" alt="screen shot 2017-01-06 at 20 37 06" src="https://cloud.githubusercontent.com/assets/1308971/21732131/ec4425de-d44f-11e6-829d-b888a11b8b02.png">